Web/react-spa/src/components/basket/BasketItemExcerpt.tsx

```diff
@@ -0,0 +1,13 @@
+import BasketItem from "features/basket/basketItem"
+
+export function BasketItemExcerpt({ item }: { item: BasketItem }) {
```

Comment:
for cleanliness: 
`export type BasketItemProps = {
item: BasketItem
}

export function BasketItem({item}: BasketItemProps)`

you should always declare props type. you may not export it if not required